test/test_serializers/test_prettyxml.py

```diff
@@ -43,19 +43,19 @@ def _assert_equal_graphs(g1, g2):
 
 def _mangled_copy(g):
     "Makes a copy of the graph, replacing all bnodes with the bnode ``_blank``."
-    gcopy = ConjunctiveGraph()
+    gcopy = Dataset()
 
     def isbnode(v):
         return isinstance(v, BNode)
 
-    for s, p, o in g:
+    for s, p, o, c in g:
         if isbnode(s):
             s = _blank
         if isbnode(p):
             p = _blank
         if isbnode(o):
             o = _blank
-        gcopy.add((s, p, o))
+        gcopy.add((s, p, o, DATASET_DEFAULT_GRAPH_ID if c is None else c))
```

Comment:
I know this is still in draft, just thinking, would it not be better to associate None with the default Graph ID? That would make the interface a bit easier to use I think.